indigo_app/views/places.py

```diff
@@ -114,3 +117,31 @@ def get_context_data(self, **kwargs):
         context['work_n_amendments_json'] = json.dumps(amendments)
 
         return context
+
+
+class PlaceActivityView(PlaceViewBase, MultipleObjectMixin, TemplateView):
+    model = None
+    slug_field = 'place'
+    slug_url_kwarg = 'place'
+    template_name = 'place/activities.html'
+    tab = 'activities'
+
+    object_list = None
+    page_size = 20
+    js_view = ''
+
+    def get_context_data(self, **kwargs):
+        context = super(PlaceActivityView, self).get_context_data(**kwargs)
+
+        all_activities = Action.objects.all()
+        activities = [act for act in all_activities if act.data and act.data['place_code'] == self.place.code]
```

Comment:
We need to get the database filter working, this will rapidly become unsustainable.